docs/user/SDarnRead.md

```diff
@@ -0,0 +1,53 @@
+# Reading in DMap structured SuperDARN data files
+---
+
+Data Map (DMap) is a binary self-describing format that was developed by Rob Barnes. 
+This format is currently the primary format used by SuperDARN. 
+For more information on DMap please see [RST Documentation](https://radar-software-toolkit-rst.readthedocs.io/en/latest/).
+Types of files used by SuperDARN which are usually accessed in DMap format are:
+- IQDAT
+- RAWACF
+- FITACF
+- GRID/GRD
+- MAP
+
+This tutorial will focus on reading in DMap structured files using Pydarn, including how to access common data fields.
+
+## Reading with SDarnRead
+
+The basic code to read in a DMap structured file is as follows:
+```python
+import pydarn
+
+file = "path/to/file"
+SDarn_read = pydarn.SDarnRead(file)
```

Comment:
Is there a bit of branch confusion here?  I checked out this PR and `SDarnRead` is `DarnRead`.

Reply:
Also, a question.  Is it appropriate to use camel case for variables that are classes?  I thought one was only supposed to use this for the class definition itself.

Reply:
https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html
It follows the python convention for naming things. 

If you have any other code-related questions please ask on our team chat or slack :) I would like to keep this branch focused on documentation.

Reply:
@mardet987 just merged develop into doc/user you can now test things.